### PR TITLE
tests: Use python3 for lockf-n.py

### PR DESCRIPTION
**Pull request by mkrajnak**

### CODE REVIEW

**Comment by smcv** [comment]
bot, test pull request

**Comment by smcv** [comment]
We have a similar patch in Debian, which also makes the obvious change in `demos/userns-block-fd.py`. It might make sense to upstream that too, now that Python 2 is officially EOL.

**Comment by smcv** [comment]
@cgwalters, @alexlarsson Is there documentation for how to operate the rh-atomic-bot, and would it be possible to make it drop a link when it comments? I'm sure Red Hat / Project Atomic people always remember how to use it anyway, but this is the only containers package I'm a reviewer on, so I'm never sure whether I'm doing it right.

**Comment by alexlarsson** [comment]
I thought rh-atomic-bot was discontinued. @cgwalters ?

**Comment by alexlarsson** [comment]
bot, test pull request

**Comment by alexlarsson** [comment]
/usr/bin/env: python3: No such file or directory
I guess we need python3 in the CI

File: tests/test-run.sh [hunk 1]
```diff
@@ -226,7 +226,7 @@ fi
 # Test --die-with-parent
 
 cat >lockf-n.py <<EOF
-#!/usr/bin/env python
+#!/usr/bin/env python3
```
Comment (smcv) [inline tests/test-run.sh]: `centos7` build now fails:

```
+ ./lockf-n.py ./lock nowait
/usr/bin/env: python3: No such file or directory
```

The `ci/papr.sh` probably now needs to `yum -y install ... python3` or similar.

**Comment by smcv** [comment]
> I thought rh-atomic-bot was discontinued

Not yet, apparently. See #335.

**Comment by mkrajnak** [comment]
> We have a similar patch in Debian, which also makes the obvious change in `demos/userns-block-fd.py`. It might make sense to upstream that too, now that Python 2 is officially EOL.

Would you like to have it included in this PR ?

**Comment by smcv** [comment]
> Would you like to have it included in this PR ?

If the project is going to require `python3` for `make check` then I think it would make sense to be consistent about using `python3` everywhere.

Here's the patch I've been applying in the Debian packaging for the last 2 years: https://salsa.debian.org/debian/bubblewrap/blob/debian/master/debian/patches/debian/Use-Python-3-for-test-demo-code.patch

I initially did it as a Debian-specific change rather than upstreaming it, because Debian has been more aggressive about deprecating Python 2 than a lot of upstream projects - but now that Python 2 is officially EOL I think it makes sense to make the change upstream.

**Comment by alexlarsson** [comment]
Yeah, i think its best to change it everywhere.

File: tests/test-run.sh (hunk 1 above)
Comment (smcv) [reply tests/test-run.sh]: Please change `ci/papr.sh` to add `python3` to the list of packages on the `yum` command-line, something like this:

```diff
     yum -y install git autoconf automake libtool make gcc redhat-rpm-config \
         libcap-devel  'pkgconfig(libselinux)' 'libxslt' 'docbook-style-xsl' \
-        lib{a,ub,t}san /usr/bin/eu-readelf rsync
+        lib{a,ub,t}san /usr/bin/eu-readelf rsync python3
```

That should hopefully be enough to fix the CI failure.

(You'll be doing this blindly - I don't know how to run the CI myself - but it's a simple enough change to make that way with reasonable confidence.)

**Review by smcv — APPROVED** [review]

**Pull request merged**